cabot/metricsapp/api/elastic.py

```diff
@@ -1,11 +1,54 @@
 from elasticsearch import Elasticsearch
+from enum import Enum
+
+
+class SupportedMetrics(Enum):
```

Comment:
How does this being a subclass of `Enum` help? It will work with `SupportedMetrics(object)` too.